### PR TITLE
Update harfbuzz fuzzer config to include new hb-subset-fuzzer target.

### PR DESCRIPTION
**Pull request by garretrieger**

### CODE REVIEW

File: projects/harfbuzz/hb-subset-fuzzer.options [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
+timeout = 5
```
Comment (Dor1s) [inline projects/harfbuzz/hb-subset-fuzzer.options]: Are you sure about that? i.e. any input taking more than 5 seconds to process would be reported as a timeout. By default we use 25 seconds.

File: projects/harfbuzz/build.sh [hunk 2]
```diff
@@ -26,13 +26,17 @@ export CXXFLAGS="$CXXFLAGS -fno-sanitize=function,vptr"
 ./configure
 make clean
 make -j$(nproc) V=1 all
-make -C src V=1 fuzzing
+make CPPFLAGS="-DHB_NO_VISIBILITY" -C src V=1 fuzzing
 
 # Build the fuzzer.
 $CXX $CXXFLAGS -std=c++11 -Isrc \
-    ./test/fuzzing/hb-fuzzer.cc -o $OUT/hb-fuzzer \
+    ./test/fuzzing/hb-shape-fuzzer.cc -o $OUT/hb-shape-fuzzer \
```
Comment (Dor1s) [inline projects/harfbuzz/build.sh]: Would it be possible to move these upstream, and have here only something like "make fuzzers" or whatever build system is used. We propose it to every project as part of the ideal integration (https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md).

Not a blocker for this PR, but would be a great improvement in future.

Comment (garretrieger) [reply projects/harfbuzz/build.sh]: Acknowledged, I'll plan to do this separate from this PR.

**Comment by Dor1s** [comment]
Merged, thanks!

**Pull request merged**